plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClientModule.java

```diff
@@ -48,24 +50,53 @@ public class OracleClientModule
     @Override
     public void configure(Binder binder)
     {
-        binder.bind(JdbcClient.class).annotatedWith(ForBaseJdbc.class).to(OracleClient.class).in(Scopes.SINGLETON);
+        binder.bind(JdbcClient.class).annotatedWith(ForBaseJdbc.class).to(OracleClient.class).in(SINGLETON);
         bindSessionPropertiesProvider(binder, OracleSessionProperties.class);
         configBinder(binder).bindConfig(OracleConfig.class);
         newOptionalBinder(binder, Key.get(int.class, MaxDomainCompactionThreshold.class)).setBinding().toInstance(ORACLE_MAX_LIST_EXPRESSIONS);
-        newSetBinder(binder, ConnectorTableFunction.class).addBinding().toProvider(Query.class).in(Scopes.SINGLETON);
+        newSetBinder(binder, ConnectorTableFunction.class).addBinding().toProvider(Query.class).in(SINGLETON);
+        binder.install(new HideDetailsModule());
     }
 
-    @Provides
-    @Singleton
-    @ForBaseJdbc
-    public static ConnectionFactory connectionFactory(BaseJdbcConfig config, CredentialProvider credentialProvider, OracleConfig oracleConfig, OpenTelemetry openTelemetry)
-            throws SQLException
+    private static class HideDetailsModule
+            extends PrivateModule
     {
-        Properties connectionProperties = new Properties();
-        connectionProperties.setProperty(OracleConnection.CONNECTION_PROPERTY_INCLUDE_SYNONYMS, String.valueOf(oracleConfig.isSynonymsEnabled()));
-        connectionProperties.setProperty(OracleConnection.CONNECTION_PROPERTY_REPORT_REMARKS, String.valueOf(oracleConfig.isRemarksReportingEnabled()));
+        @Override
+        protected void configure()
+        {
+            expose(ConnectionFactory.class).annotatedWith(ForBaseJdbc.class);
+            expose(ConnectionFactory.class).annotatedWith(ForRetryingConnectionFactory.class);
+//            install(conditionalModule(
```

Comment:
Remove commented out code

Reply:
Sorry. It was there only because of the other comment from @kokosing. Now removed